registry/storage/blobstore.go

```diff
@@ -188,10 +349,50 @@ func (bs *blobStatter) Stat(ctx context.Context, dgst digest.Digest) (distributi
 	}, nil
 }
 
+// Clear deletes blob's data with or without its parent directory depending on
+// bs' configuration.
 func (bs *blobStatter) Clear(ctx context.Context, dgst digest.Digest) error {
-	return distribution.ErrUnsupported
+	var (
```

Comment:
I wonder myself. The `Clear` function is already a part of its interface. What other use case can it have other than delete. Shall I move the functionality to `Delete` and leave the `Clear()` as it is now?